protocol/messenger_communities.go

```diff
@@ -3949,6 +3954,10 @@ func (m *Messenger) InitHistoryArchiveTasks(communities []*communities.Community
 			for _, filter := range filters {
 				topics = append(topics, filter.ContentTopic)
 			}
+			// adding the content-topic used for member updates.
+			// since member updates would not be too frequent i.e only addition/deletion would add a new message,
+			// this shouldn't cause too much increase in size of archive generated.
+			filters = append(filters, m.transport.FilterByChatID(c.MemberUpdateChannelID()))
```

Comment:
It is, but sending to both old and new will double bandwidth usage which is already high. 
Hence the plan is to use an existing content-topic.

Any risks i think should be identified during review or QA. Do you see any obvious challenges?

Reply:
right now the plan is to match the current filter criteria to topics. 

Most probably in the next phase would involve analysis and figuring out how to improve filter criteria for various optimizations. At that time a change to the way content-topics are assigned may occur and might need an upgrade.

So, anyways we will have to do an upgrade in future irrespective of whether we reuse existing content-topic or not.
maybe you can elaborate further.

Reply:
> m thinking that using `MemberUpdateChannelID ` as the common topic is likely a short term solution

yep, that is the idea. 



> This would mean we need make a new break change in the future.

Yep, i think that is unavoidable or we will have to duplicate bandwidth usage(probably once we have usage optimized we can duplicate sending msgs to old and new topics then)




> I'm ok with current solution, since bandwidth is more important now, it actually makes more sense.

thanks, that was the idea behind taking this approach.